hack/test-e2e-local.sh

```diff
@@ -38,6 +38,7 @@ if [[ "$1" != "operator" ]]; then
     e2e-default.local
     e2e-upgrade-node.local
     e2e-upgrade-zone.local
+    e2e-g-upgrade.local
```

Comment:
```suggestion
    e2e-upgrade.local
```

Maybe we should rename the other two `e2e-upgrade-{node,zone}` to `e2e-update-...` instead.